cmd/catalyst-node/catalyst-node.go

```diff
@@ -459,9 +462,20 @@ func protocol(r *http.Request) string {
 	return "http"
 }
 
-func queryMistForClosestNode(playbackID string) (string, error) {
+func queryMistForClosestNode(playbackID, lat, lon string) (string, error) {
 	url := fmt.Sprintf("http://localhost:%s/%s", mistUtilLoadPort, playbackID)
-	resp, err := http.Get(url)
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", err
+	}
+	if lat != "" && lon != "" {
+		req.Header.Set("X-Latitude", lat)
+		req.Header.Set("X-Longitude", lon)
+	} else {
+		glog.Warningf("Incoming request missing X-Latitude/X-Longitude, response will not be geolocated")
+	}
+	client := &http.Client{}
+	resp, err := client.Do(req)
```

Comment:
Will this automatically use the lat/lon info or do we need to use the `--geo` flag to add weight to MistUtilLoad's scoring logic?